bin/lint

```diff
@@ -11,7 +11,6 @@ _docker_run() {
     local docker_volume_opts=""
   fi
   docker run --rm -it \
-    -e OC_LOGIN=false \
```

Comment:
I think you must also change the `.circleci/config.yml` with this change.

Reply:
Done.